dataservice/api/genomic_file/schemas.py

```diff
@@ -1,17 +1,29 @@
 from marshmallow_sqlalchemy import field_for
 
 from dataservice.api.genomic_file.models import GenomicFile
-from dataservice.api.common.schemas import BaseSchema, IndexdFileSchema
+from dataservice.api.common.schemas import (BaseSchema, IndexdFileSchema,
+                                            AVAILABILITY_ENUM)
 from dataservice.api.common.custom_fields import PatchedURLFor
 from dataservice.extensions import ma
+from dataservice.api.common.validation import enum_validation_generator
+
+DATA_TYPE_ENUM = {'Submitted Aligned Reads',
```

Comment:
I'm not sure if having the defaults on the data types makes sense:
- Not Applicable
- Not Available
- Not Reported
- Not Allowed To Collect

Reply:
Also, are `Submitted` types just what we get from the source? Are there `harmonized` types too?

Reply:
I think in GDC the harmonized files just had the `Submitted` removed. But maybe we want to be more specific and call it `Harmonized Aligned Reads`, etc?

Reply:
in regards to defaults on data_type, for null values we at least have choice to fill. yeah, i did see for few columns where some of these defaults are unnecessary. Once, we get real data in some of these enums will be refactored